### PR TITLE
Do not log if time remaining is <= 0

### PR DESCRIPTION
**Pull request by bsquizz**
Fix for log messages like this:
```
2023-02-21 18:06:38 [    INFO] [thread-30 (wait_for_ready)] [clowdapp/storage-broker] waiting -2sec longer
```

### CODE REVIEW

File: ocviapy/__init__.py [hunk 1]
```diff
@@ -664,14 +664,16 @@ def check_ready(self):
         return False
 
     def _check_with_periodic_log(self):
+        current_time = time.time()
```
Comment (Victoremepunto) [inline ocviapy/__init__.py:667]: I'd add an explicit `int` conversion here for the comparison's sake

File: ocviapy/__init__.py [hunk 2]
```diff
@@ -664,14 +664,16 @@ def check_ready(self):
...
+
         if self.check_ready():
             return True
 
-        if time.time() > self._time_last_logged + 60:
+        if current_time > self._time_last_logged + 60:
             self._time_remaining -= 60
```
Comment (Victoremepunto) [inline ocviapy/__init__.py:673]: another `int` explicit conversion

also what's the `60` hardcoded value about , I see it added and then subtracted from _time_remaining ? if it's some sort of "security margin" ? if so, I'd strongly recommend to explicitly set so in the form of a variable with name that indicates so for better clarity 
```suggestion
        if current_time > int(self._time_last_logged + 60):
            self._time_remaining -= 60
```

**Comment by bsquizz** [comment]
@Victoremepunto check out the latest changes.

`60` sec is the interval that we print a log message. So there is a `wait_for` loop running which is the function actually in charge of waiting for the condition to become `True` and fail after a certain timeout. But, we log a message every 60 seconds while this wait for loop is running to give users an update that `we are still waiting for N seconds longer`. I am storing the log message interval in a variable now -- hopefully this makes it easier to understand.

Also, the two times I store `time.time` I am converting it to a variable now. I think this covers all the situations where the var could be set as a float -- so these variables should always have `int` values now.

**Comment by Victoremepunto** [comment]
> @Victoremepunto check out the latest changes.
> 
> `60` sec is the interval that we print a log message. So there is a `wait_for` loop running which is the function actually in charge of waiting for the condition to become `True` and fail after a certain timeout. But, we log a message every 60 seconds while this wait for loop is running to give users an update that `we are still waiting for N seconds longer`. I am storing the log message interval in a variable now -- hopefully this makes it easier to understand.
> 
> Also, the two times I store `time.time` I am converting it to an int now. I think this covers all the situations where the var could be set as a float -- so these variables should always have `int` values now.

I think it's more clear now, thanks for the changes. I can't explicitly approve, I'd do so if I could.

My only remaining comment would be adding tests along this change (as usual) but I don't see any on the project so ... :shrug:

**Comment by bsquizz** [comment]
Tested it using a local bonfire virtual env

**Pull request merged**